app/src/main/java/com/infomaniak/drive/ui/addFiles/NewFolderFragment.kt

```diff
@@ -81,8 +82,9 @@ class NewFolderFragment : Fragment() {
         }
     }
 
-    private fun initDropBoxFolder(drive: Drive?) {
-        binding.dropBox.setOnClickListener {
+    private fun initDropBoxFolder(drive: Drive?) = binding.dropBox.apply {
```

Comment:
I like to put a `:unit` when an apply return something, and the method should not return anything